_rules/explicit-SVG-image-has-name-7d6734.md

```diff
@@ -141,7 +141,7 @@ The `circle` element has an explicit role of `graphics-symbol` but does not have
 
 #### Inapplicable Example 1
 
-Both the `svg` and `circle` elements do not have an explicit role.
+Neither the `svg` element nor the `circle` element has an [explicit role][].
```

Comment:
Make it more clear and consistent with other example i.e. inapplicable example 3.
```suggestion
Neither the `svg` element nor the `circle` element has any of the [explicit role][] of  `img`, `graphics-document`, `graphics-symbol`.
```